mlir/lib/Dialect/Quant/IR/TypeParser.cpp

```diff
@@ -273,6 +290,28 @@ static Type parseUniformType(DialectAsmParser &parser) {
     return nullptr;
   }
 
+  // Quantile list
+  if (isQuantile) {
+    if (parser.parseLBrace()) {
+      return nullptr;
+    }
+
+    do {
+      quantiles.resize(quantiles.size() + 1);
```

Comment:
nitpick: `quantiles.emplace_back()` would be better here since the growth factor of a vector would be used while currently you'd grow by 1 element N times (making this O(N * M) procedure).

Reply:
good point, I updated it